### PR TITLE
Add AOCODARCF4V3

### PR DESCRIPTION
**Pull request by 634296207**
Add AOCODARCF4V3

### CODE REVIEW

**Comment by aocodarc** [comment]
a) OK, we use AOCODARCF4V3 for the target, keep target naming format consisitency with F4V2
b) Please do self test. Make sure self-test is passed, and add comment below

```
F405V3 Flash V2 self test passed.
```

File: src/main/target/AOCODARCF4V3/target.h [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+#define M25P16_SPI_BUS                  BUS_SPI3
+#define M25P16_CS_PIN                   PC0
+
+#define ENABLE_BLACKBOX_LOGGING_ON_SPIFLASH_BY_DEFAULT
+
+// *************** PINIO ***************************
+#define USE_PINIO
+#define USE_PINIOBOX
+#define PINIO1_PIN                  PC5  // VTX power switcher
+#define PINIO2_PIN                  PA14  //
```
Comment (lida2003) [inline src/main/target/AOCODARCF4V3/target.h:145]: Add comment for Bluetooth switcher please!

Comment (lida2003) [inline src/main/target/AOCODARCF4V3/target.h]: Add comment for Bluetooth switcher, like VTX power switcher, please!

File: src/main/target/AOCODARCF4V3/target.h [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+//************ LEDSTRIP *****************
+#define USE_LED_STRIP
+#define WS2811_PIN                      PB1
+
+// ******* FEATURES ********
+#define DEFAULT_RX_TYPE                 RX_TYPE_SERIAL
+#define SERIALRX_UART                   SERIAL_PORT_USART2
+#define SERIALRX_PROVIDER               SERIALRX_SBUS
```
Comment (lida2003) [inline src/main/target/AOCODARCF4V3/target.h]: I thought SERIALRX_SBUS / SERIALRX_CRSF is used for our AOC 7 inch PNP copters.

1. SERIALRX_SBUS for HD
2. SERIALRX_CRSF  for analog

I didn't do the math. If people like to use DJI RC more than ELRS then SERIALRX_SBUS , or SERIALRX_CRSF   should be set by default.

**Review by lida2003 — CHANGES_REQUESTED** [review]

**Comment by 634296207** [comment]
1.Add Bluetooth comment
2.Use the crsf protocol
2.Turn on the led light belt function

**Comment by lida2003** [comment]
Default settings, please keep consistent with https://github.com/Aocoda-RC/unified-targets/pull/2

**Comment by 634296207** [comment]
> Default settings, please keep consistent with [Aocoda-RC/unified-targets#2](https://github.com/Aocoda-RC/unified-targets/pull/2）
I looked at someone else's code and couldn't find the Settings for dsoht600

**Comment by 634296207** [comment]
I looked at someone else's code and couldn't find the Settings for dsoht600

**Comment by lida2003** [comment]
> I looked at someone else's code and couldn't find the Settings for dsoht600

OK, i see.

Can you cleanup the PR, as there are quite a lot of irrelevant commits?

**Pull request closed without merging**